pkg/datasource/compat/wrapper.go

```diff
@@ -134,7 +134,14 @@ func WrapAccessors(source datasource.DataSource, mntnsidAccessor datasource.Fiel
 		return nil, err
 	}
 
-	ev.nodeAccessor, err = k8s.AddSubField("node", api.Kind_String, datasource.WithTags("kubernetes"))
+	ev.nodeAccessor, err = k8s.AddSubField("node",
+		api.Kind_String,
+		datasource.WithTags("kubernetes"),
+		datasource.WithAnnotations(map[string]string{
+			datasource.TemplateAnnotation: "node",
+		}),
+		datasource.WithOrder(-31),
```

Comment:
How was it working before?

Reply:
Which part? The `k8s.node` fields was already there, I am just defining an order so it comes to the front or am I missing something?

Reply:
I mean, IIRC, `k8s.node` was printed by first in the past. So, what did break it that made necessary to specify the order for it?

Reply:
IIUC it was like that since refactoring, initially `k8s` [columns were hidden ](https://github.com/inspektor-gadget/inspektor-gadget/issues/3014)so we didn't notice it? They were enabled with: https://github.com/inspektor-gadget/inspektor-gadget/pull/3136#issue-2392550863 and PR output confirms it was the case.